src/Microsoft.DotNet.Wpf/src/WindowsBase/LibraryAssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("WindowsBase.Tests, PublicKey=00000000000000000400000000000000")]
```

Comment:
Is there a reason for creating additional file instead of using established `OtherAssemblyAttrs`?

Reply:
It is included in the `-ref` project.

Reply:
Ah, I've overlooked we're referencing the same file here for ref. Makes sense, thanks.

Makes me wonder whether it's a correct approach to reference it though but it doesn't really matter.